### DISCUSSION TITLE
Designing Success Stories Card - Seeking Feedback and Ideas

### DISCUSSION THREAD

**Discussion by VenkaSri** [TheOdinProject/theodinproject#4187] [Ideas]
I've been working on improving the design of success story cards shown in the main page, and I'd love to hear your thoughts and suggestions. Currently, success stories from home page looks like this:

<img width="600" alt="Screenshot 2023-09-24 at 11 04 36 AM" src="https://github.com/TheOdinProject/theodinproject/assets/53845532/993a4cf2-f15f-4a3f-8b77-9459207aef61">

I've designed a couple of alternative card designs: 
<img width="600" alt="Screenshot 2023-09-24 at 12 01 14 PM" src="https://github.com/TheOdinProject/theodinproject/assets/53845532/0920a5ee-88e7-4b02-8a04-9e62e04fb4d6">

Would love your inputs @ManonLef @KevinMulhern

**Comment by KevinMulhern** [comment]
Thanks for taking the time to write this up @VenkaSri. Cards are great for structure and separation but I'm not sure if they work well naturally for this section.

If we were to redesign this, I'd like to have a design that feels more hand crafted and personable. Like the person put the success story on the page themselves. You'll often see sites use a design that looks like tweets or other [types of social media posts](https://tailwindui.com/components/marketing/sections/testimonials#component-1d4641d9b778e6e47e5cb664d4d8a683) to achieve that kind of thing.

**Reply by toduyemi** [reply]
I think the card idea is great (I vote the 2nd one), but with the caveat of instead of showcasing the person (picture of their face). You showcase one of their more advanced well done Odin projects. It adds more weight and veracity to the success story if visitors can see what that person created that led to them being hireable. They'll also be capable of that if they were to embark on the curriculum.
